include/sound/sof/stream.h

```diff
@@ -83,10 +83,10 @@ struct sof_ipc_stream_params {
 	uint16_t sample_valid_bytes;
 	uint16_t sample_container_bytes;
 
-	/* for notifying host period has completed - 0 means no period IRQ */
 	uint32_t host_period_bytes;
+	uint16_t no_stream_position; /**< 1 means don't send stream position */
```

Comment:
fix typo in commit message: host_period_bytes ~~preseves~~ preserves its
original value.

Reply:
you haven't fixed the 'preserves' typo and added another in the commit title: it's ASoC: